doc/source/setup/macos.rst

```diff
@@ -63,6 +63,8 @@ instructions::
 
     brew info mosquitto influxdb grafana | grep Caveats -A8
 
+.. todo:: Add walkthrough how to install CrateDB on macOS.
+
```

Comment:
Probably, it would be about following [^1], and acquiring the corresponding tarball from [^2]. 

[^1]: https://crate.io/docs/crate/tutorials/en/latest/basic/index.html#ad-hoc-unix-macos
[^2]: https://cdn.crate.io/downloads/releases/cratedb/x64_mac/

Reply:
Maybe just refer to the Docker instructions instead.